bin/crystal

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env sh
+#!/bin/sh
```

Comment:
@bew this PR is basically a follow up to this one: https://github.com/crystal-lang/crystal/pull/3809